fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -2723,26 +2817,19 @@ ACTOR void setupAndRun(std::string dataFolder,
 	state bool allowCreatingTenants = testConfig.allowCreatingTenants;
 
 	if (!SERVER_KNOBS->SHARD_ENCODE_LOCATION_METADATA) {
-		testConfig.storageEngineExcludeTypes.push_back(5);
-	}
-
-	// The RocksDB engine is not always built with the rest of fdbserver. Don't try to use it if it is not included
-	// in the build.
-	if (!rocksDBEnabled) {
-		testConfig.storageEngineExcludeTypes.push_back(4);
-		testConfig.storageEngineExcludeTypes.push_back(5);
+		testConfig.storageEngineExcludeTypes.insert(SimulationStorageEngine::SHARDED_ROCKSDB);
```

Comment:
exclude `SimulationStorageEngine::ROCKSDB` if rocks is not enabled.

Reply:
`SimulationStorageEngine::ROCKSDB` can work with `SHARD_ENCODE_LOCATION_METADATA` off.